tests/layer_tests/py_frontend_tests/test_torch_frontend.py

```diff
@@ -464,6 +464,87 @@ def forward(self, x):
         "Parameter", "Sin", "Tan", "Add", "Result"]
 
 
+def test_inlined_extension():
+    import openvino as ov
+    from openvino.frontend.pytorch import inlined_extension
+    import numpy as np
+
+    rng = np.random.default_rng(42)
+
+    @inlined_extension
+    def numpy_cos(x):
+        # numpy is not captured by tracing, so we use a custom extension
+        return torch.from_numpy(np.cos(x.numpy(force=True)))
+
+    class ModelWithModule(torch.nn.Module):
+        def __init__(self):
+            super().__init__()
+            self.relu_module = torch.nn.ReLU()
+
+        def forward(self, x):
+            x = x.to(torch.float32)
+            return numpy_cos(x) + self.relu_module(x)
+
+    model = ModelWithModule()
+
+    example = torch.from_numpy(rng.random([100], dtype=np.float32))
+    converted_model = ov.convert_model(model, example_input=(example,))
```

Comment:
will inlined extension work for export scenario?

Reply:
It is not tested in export scenario, most likely additional work is required to make it work there